static/app/views/settings/organizationDeveloperSettings/sentryFunctionDetails.tsx

```diff
@@ -104,20 +139,43 @@ function SentryFunctionDetails(props: Props) {
             </PanelBody>
           </Panel>
         </Form>
+        {sentryFunction && (
+          <Button
+            onClick={handleDelete}
+            title={t('Delete Sentry Function')}
+            aria-label={t('Delete Sentry Function')}
+            type="button"
+            priority="danger"
+          >
+            Delete Sentry Function
```

Comment:
@vpandiarajan20 So this is something we should use the translation function `t` for to ensure that if the user has another language they can show it in that language.